invisible_cities/cities/components_test.py

```diff
@@ -191,33 +193,50 @@ def test_copy_mc_info_repeated_event_numbers(ICDATADIR, config_tmpdir):
 
 
 def test_mcsensors_from_file_fast_returns_empty(ICDATADIR):
+    rate = 0.5
     file_in = os.path.join(ICDATADIR, "nexus_new_kr83m_fast.newformat.sim.h5")
-    sns_gen = mcsensors_from_file([file_in], 'new', -7951)
+    sns_gen = mcsensors_from_file([file_in], 'new', -7951, rate)
     first_evt = next(sns_gen)
     assert first_evt[ 'pmt_resp'].empty
     assert first_evt['sipm_resp'].empty
 
 
 def test_mcsensors_from_file_correct_yield(ICDATADIR):
     evt_no         =    0
-    timestamp      =    0
+    # timestamp      =    0
```

Comment:
Don't leave code commented out. remove the line if you don't need to use it anymore